catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/Relationship.java

```diff
@@ -1,111 +0,0 @@
-/*
- *  Copyright 2021 Collate
- *  Licensed under the Apache License, Version 2.0 (the "License");
- *  you may not use this file except in compliance with the License.
- *  You may obtain a copy of the License at
- *  http://www.apache.org/licenses/LICENSE-2.0
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
- */
-
-package org.openmetadata.catalog.jdbi3;
-
-/**
- * This enum captures all the relationships between Catalog entities Note that the relationship from is a Strong entity
```

Comment:
@sureshms  I'm moving all of the comments here to documentation (also part of this PR)

Reply:
We should add some pictorial representations of relationships in documents, sometime in the future.